xapian-core/include/xapian/weight.h

```diff
@@ -471,10 +471,12 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
      * Implementing support for more normalizations of each type would require
      * extending the backend to track more statistics.
      */
-    explicit TfIdfWeight(double slope, double delta, const std::string &normalizations);
+    explicit TfIdfWeight(const std::string &normalizations);
+
+    explicit TfIdfWeight(const std::string &normalizations, double slope, double delta);
 
     TfIdfWeight()
-	: param_slope(0.2), param_delta(1.0), normalizations("ntn")
+	: normalizations("ntn")
```

Comment:
If I construct `TfIdfWeight("PPP")`, should slope and delta be `0.0` or `0.2` and `1.0` respectively? What's more useful?